docs/pages/guides/meshery-metrics.md

```diff
@@ -1,99 +1,195 @@
 ---
 layout: default
 title: Using Metrics in Meshery
-description: How to connected and use Prometheus and*Grafana*metrics in Meshery
+description: How to connect and use Prometheus and Grafana metrics in Meshery
 permalink: guides/meshery-metrics
 type: Guides
 ---
 
-In order to generate performance test reports of service meshes and their workloads, Meshery uses **Grafana** and/or **Prometheus** as visualization and metrics systems, respectively. This document outlines the requirements necessary for Meshery to connect to these systems. Steps vary depending upon the service mesh and its configuration.
+# Connect and use metrics in Meshery
 
-* TOC
-{:toc}
+Meshery provides performance reports, including performance test results, node resource metrics etc. so that operators may easily understand the overhead of their service mesh's control plane and data plane in context of the overhead incurred on nodes running within the cluster. In order to generate performance test reports of service meshes and their workloads, Meshery uses `Grafana` and/or `Prometheus` as visualization and metrics systems, respectively. This guide outlines the requirements necessary for Meshery to connect to these systems. The steps may vary depending upon the service mesh and its configuration.
```

Comment:
We'll want to generally reserve backticks (code tags) for shell, code, yaml, the name of a CLI (because it's shell-related) and plainly use capitalization for project names.